enatega-multivendor-admin/src/components/Rider/Rider.jsx

```diff
@@ -223,6 +223,19 @@ function Rider(props) {
                     : ''
                 ]}
               />
+              {usernameError === false && (
+                <Typography
+                  variant="p"
+                  style={{
+                    color: 'red',
+                    fontSize: '12px',
+                    float: 'left',
+                    marginTop: '5px',
+                    marginLeft: '10px'
+                  }}>
+                  {t('Username cannot contain spaces')}
```

Comment:
Instead of writing a hard-coded message, make it dynamic and display the error we are actually getting from usernameError. Right now, if the user does not enter anything in the username field, it will still show "space not allowed" error instead of showing username cannot be empty or simply red border.

Also try to make key in translation files for the error message

Reply:
sure i will do it !